helper/impls/helper-impl-14/src/main/java/org/apache/avro/io/Avro14BinaryDecoderConfigurer.java

```diff
@@ -0,0 +1,15 @@
+/*
+ * Copyright 2020 LinkedIn Corp.
+ * Licensed under the BSD 2-Clause License (the "License").
+ * See License in the project root for license information.
+ */
+
+package org.apache.avro.io;
+
+public class Avro14BinaryDecoderConfigurer {
```

Comment:
the convention is to call these classes *AccessUtil - so Avro14BinaryDecoderAccessUtil?
also - can be static